osu.Game/Rulesets/Edit/Layers/Selection/SelectionOverlay.cs

```diff
@@ -60,6 +68,27 @@ private void load(OsuColour colours)
             Position = topLeft;
         }
 
+        protected override bool OnMouseDown(InputState state, MouseDownEventArgs args) => true;

```

Comment:
I'd probably add

```csharp
public override bool ReceiveMouseInputAt(Vector2 screenSpacePos) => overlays.Any(o => o.IsHovered);
```

to this class

Reply:
Nah, because you should be able to select a group of objects and click-drag anywhere in the selection box to move them. It feels super bad otherwise - might as well not have the box in the first place.

Reply:
hmm, this makes it very awkward to make new selections. we may want to reconsider the visuals of the box to make this not the case, because i believe we want the behaviour i mention instead (give it a try, it definitely feels better when editing, and is how the stable editor works).